starlight_help/src/content/docs/bots-overview.mdx

```diff
@@ -42,31 +42,32 @@ The **bot type** determines what the bot can do.
 | Incoming webhook | Limited to only sending messages into Zulip                        | Automated notifications into Zulip                              |
 | Outgoing webhook | Generic bot that also receives new messages via HTTP post requests | Third party integrations, most custom bots                      |
 
+A **generic** bot is like a normal Zulip user account that cannot log in via a
+browser. There's a handful of actions no bots can't take, including creating
```

Comment:
"actions no bots can't take"

Reply:
I don't love this description. I'd say "A generic bot acts like a normal Zulip user that can only access Zulip via the API." seems cleaner